package.json

```diff
@@ -14,14 +14,19 @@
     "url": "git@github.com:botify-labs/react-classmap.git"
   },
   "license": "MIT",
+  "devEngines": {
+    "node": "4.x"
+  },
   "dependencies": {
-    "react": "^0.13.3"
+    "react": "^0.14.0",
```

Comment:
should it be `"~0.14.0"`? as it might only be compatible with `0.14.x`
should react dependencies be in `peerDependencies`?